ansible-ipi-install/roles/installer/templates/install-config.j2

```diff
@@ -106,9 +106,9 @@ platform:
 {% endif %}
 {% endif %}
 {% if ((release_version[0]|int == 4) and (release_version[2]|int >= 6)) %}
-{% if 'boot_mode' in hostvars[host] %}
-        bootMode: {{ hostvars[host]['boot_mode'] }}
-{% endif %}
+{% if bootmode == 'legacy' %}
+        bootMode: legacy
+{% endif %} 
```

Comment:
Can you add the same piece of code for the install-config-virtualmedia.j2 file as well?

Reply:
@rlopez133 I couldn't find that file in that path so I ignored ? could you point me to the file please.

Reply:
@mukrishn are you looking for the below file?
 https://github.com/openshift-kni/baremetal-deploy/blob/master/ansible-ipi-install/roles/installer/templates/install-config.j2 , this should be the path of that template file.

Reply:
@DirectedSoul1 I am looking for this template file - `install-config-virtualmedia.j2`

Reply:
I'm guessing you don't see it because @smalleni hasn't rebased with the newest changes? I added this very recently. Its in the same templates dir